examples/OcclusionSaliency.ipynb

```diff
@@ -19,81 +16,101 @@
     "- torch\n",
```

Comment:
Note "mathplotlib" should be "matplotlib" in the package list.

Reply:
I am removing this and a few other packages here because they are already depedencies of `xaitk-saliency` and should already be installed if they are running this notebook.